Gems/OpenXRVk/Code/Include/OpenXRVk/OpenXRVkSwapChain.h

```diff
@@ -66,14 +66,11 @@ namespace OpenXRVk
             XrSwapchain m_handle = XR_NULL_HANDLE;
         };
 
-        //! Assign the correct native Swapchain image based on the swapchain index and swapchain image index
+        // XR::SwapChain overrides...
```

Comment:
nit:
Make this comment doxygen friendly
```suggestion
        //! XR::SwapChain overrides...
```

Reply:
I haven't seen doing doxygen style on function override comments.